spec/acceptance/custom_repo_spec.rb

```diff
@@ -41,7 +41,7 @@ class { 'php::globals':
       it { is_expected.to be_enabled }
     end
     describe command('php --version') do
-      its(:stdout) { is_expected.to match %r{5\.6} }
+      its(:stdout) { is_expected.to match %r{8\.0} }
```

Comment:
I think [include](https://relishapp.com/rspec/rspec-expectations/v/3-9/docs/built-in-matchers/include-matcher) is a better matcher here.
```suggestion
      its(:stdout) { is_expected.to include('8.0') }
```